lit/mem/arg_style_no_extern.thorin.disabled

```diff
@@ -0,0 +1,176 @@
+// RUN: rm -f %t.ll 
```

Comment:
eh, what is this test doing? Oo

Reply:
Are you referring to the missing ` ; \` after `rm -f %t.ll`?

Currently, the test does nothing as it is disabled (issue with MacOS).
But in theory, it enumerates different possibilities of argument shapes (flat, arg, mem),
selects the current one using the c preprocessor, and calls it.

The automatic shape check is not yet implemented.

Reply:
> But in theory, it enumerates different possibilities of argument shapes (flat, arg, mem), selects the current one using the c preprocessor, and calls it.

That was the answer to my question :)